src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -53,12 +53,32 @@ public void storeMessage(Message message) {
    *     message. List is sorted by time descending.
    */
   public List<Message> getMessages(String user) {
+    return messageHelp(user);
+  }
+
+  /**
+   * Get messages posted by every user.
+   * 
+   * @return a list of messages posted by all users of the site, or an empty list if
+   *    there are no messages
+   */
+  public List<Message> getAllMessages() {
+    return messageHelp(null);
+  }
+
+  /**
+   * Helper function to make port redundant code for Message Query.
+   * 
+   * @return a list of messages p
+   */
+  private List<Message> messageHelp(String user) {
```

Comment:
I like how you pulled the logic into a helper method, but "messageHelp" is very ambiguous, could you rename the method to better explain what it does?